From 4468993bd62bfd84b5c7e9004b06cb2c7902f1c3 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Mon, 8 Jan 2024 08:45:21 +0000
Subject: [PATCH 0820/1085] ARM: dts: bcm2712-rpi-5-b: Use common LED labels

As of 6.6, the names of the labels on the Pi LEDs was swapped to match
the upstream code, i.e. led_act rather than act_led.

Apply the same change to Pi 5.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

--- a/arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts
+++ b/arch/arm/boot/dts/broadcom/bcm2712-rpi-5-b.dts
@@ -63,14 +63,14 @@
 	leds: leds {
 		compatible = "gpio-leds";
 
-		pwr_led: led-pwr {
+		led_pwr: led-pwr {
 			label = "PWR";
 			gpios = <&rp1_gpio 44 GPIO_ACTIVE_LOW>;
 			default-state = "off";
 			linux,default-trigger = "none";
 		};
 
-		act_led: led-act {
+		led_act: led-act {
 			label = "ACT";
 			gpios = <&gio_aon 9 GPIO_ACTIVE_LOW>;
 			default-state = "off";
@@ -826,10 +826,10 @@ spi10_cs_pins: &spi10_cs_gpio1 {};
 		uart0 = <&uart0>, "status";
 		wifiaddr = <&wifi>, "local-mac-address[";
 
-		act_led_activelow = <&act_led>, "active-low?";
-		act_led_trigger = <&act_led>, "linux,default-trigger";
-		pwr_led_activelow = <&pwr_led>, "gpios:8";
-		pwr_led_trigger = <&pwr_led>, "linux,default-trigger";
+		act_led_activelow = <&led_act>, "active-low?";
+		act_led_trigger = <&led_act>, "linux,default-trigger";
+		pwr_led_activelow = <&led_pwr>, "gpios:8";
+		pwr_led_trigger = <&led_pwr>, "linux,default-trigger";
 		eth_led0 = <&phy1>,"led-modes:0";
 		eth_led1 = <&phy1>,"led-modes:4";
 		drm_fb0_rp1_dsi0 = <&aliases>, "drm-fb0=",&dsi0;
